app/views/complaints/_issue_tta_report.erb

```diff
@@ -0,0 +1,23 @@
+<div> 
+  <div class="ct-tta-divider display-flex flex-justify" id="tta-activity-show-<%= issue_tta_report.tta_report_display_id %>">
+    <div><%= issue_tta_report.tta_report_display_id %></div>
+    <div>
+      <%= button_tag image_pack_tag("media/usa-icons/edit.svg"),
+        class: "ct-link-icon usa-button--unstyled js-edit-tta",
+        alt: "Edit Display ID",
```

Comment:
[issue] This isn't working properly with voiceover. I think either this should be an `aria-label` or `alt` should be on the `image_pack_tag` instead of on the `button_tag`, or maybe both of those

Reply:
ahhh yes that makes sense!